takeshape/penny
# Allow customers to checkout if they are not signed in

### Test Plan (Steps to test):

1. Go to the preview site. Make sure you're signed out.
1. Add something to your cart, attempt to checkout. It should work.
1. Pull this branch. In `.env` change this line to true `NEXT_PUBLIC_SIGNED_IN_CHECKOUT='true'`.
2. Run the site, `npm run dev`. Attempt to checkout when signed out. You should be redirected to sign in, and checkout should continue once you have.
